eZ/Publish/Core/Persistence/Legacy/Content/Handler.php

```diff
@@ -321,12 +321,67 @@ public function load($id, $version, array $translations = null)
             $this->contentGateway->loadVersionedNameData(array(array('id' => $id, 'version' => $version)))
         );
         $content = $contentObjects[0];
+        unset($rows, $contentObjects);
 
         $this->fieldHandler->loadExternalFieldData($content);
 
         return $content;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @todo
+     * REVIEW NOTE: This methods ALWAYS handles AlwaysAvailable, this would perhaps be better handled in API layer like
+     *              done on load(), thus making it optional. AND faster as it can loadContentInfoList via SPI cache.
+     *              However then we would need a argument structure like used on gateway here.
+     *              Would that be OK for loadContentList() in SPI layer??
+     *              If so it should also take version number  as part of the structure.
+     */
+    public function loadContentList(array $contentIds, array $translations): array
+    {
+        $infoList = $this->mapper->extractContentInfoFromRows(
+            $this->contentGateway->loadContentInfoList($contentIds)
+        );
+
+        if (empty($infoList)) {
+            return [];
+        }
+
+        $IdVersionPairs = [];
```

Comment:
Pascal Case detected :trollface:
Please change it to `$idVersionPairs` (and other similar variables in this block).

Reply:
I'm not following, you want `ID` instead? _(I don't see the difference :P)_

Reply:
aha, I see it now :)